modules/crowdx/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "strict": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
What is the reason for this es6 change?

Reply:
No idea, will look into it

Reply:
I set this to es5 because it does not support arrow functions. So typescript is forced to compile it to `function`s. This way it should work on more browsers.

Reply:
Fixed